setup.py

```diff
@@ -14,17 +14,18 @@
 install_requires = [
     'Faker>=3.0.0,<10',
     'graphviz>=0.13.2,<1',
-    'numpy>=1.18.0,<2',
+    'numpy>=1.19.3,<2',
```

Comment:
What is the reason for `numpy>=1.19.3`? So far I have that `sktime` requires `numpy>=1.19.0`

Reply:
This is because I removed the `sktime` dependency from SDV: `sktime 0.8.1 has requirement numpy>=1.19.3, but you have numpy 1.19.0.`
If I add ``sktime>=0.6,<1` then yes, I will roll back the change (I removed it because we didn't use it here but I think that we should attempt to keep wider range on numpy.